### PR TITLE
Add `SDK_ENVIRONMENT` to `FPTI_KEY`

### PR DESCRIPTION
**Pull request by leogedler**
- Added `SDK_ENVIRONMENT` to `FPTI_KEY` in order to use to track de environment used to execute the SDK on the logger.

### CODE REVIEW

**Review by mnicpt — APPROVED** [review]

**Comment by leogedler** [comment]
@mnicpt I have pushed new changes based on feedback received on https://github.com/paypal/paypal-smart-payment-buttons/pull/164

File: src/fpti.js [hunk 1]
```diff
@@ -56,7 +57,8 @@ export const FPTI_KEY = {
     PAY_NOW:                   ('pay_now' : 'pay_now'),
     STICKINESS_ID:             ('stickiness_id' : 'stickiness_id'),
     TIMESTAMP:                 ('t' : 't'),
-    OPTION_SELECTED:           ('optsel' : 'optsel')
+    OPTION_SELECTED:           ('optsel' : 'optsel'),
+    NATIVE:                    ('native' : 'native')
```
Comment (bluepnume) [inline src/fpti.js:61]: Why do we need this if we already have `SDK_ENVIRONMENT`?

Comment (leogedler) [reply src/fpti.js:61]: @bluepnume I added this based on your comment here: https://github.com/paypal/paypal-smart-payment-buttons/pull/164#pullrequestreview-663588016

Comment (bluepnume) [reply src/fpti.js:61]: Gotcha sorry I was talking about FPTI_STATE not FPTI_KEY. But it's fine, per discussion in other thread let's leave it out.

Comment (leogedler) [reply src/fpti.js]: @bluepnume got it, removed.

**Review by bluepnume — APPROVED** [review]

**Pull request merged**